src/components/App/App.js

```diff
@@ -57,22 +57,15 @@ const App = () => {
 
   return (
     <main className='main'>
-      <header>
+      <header className='nav-header'>
         <Link to='/'><h1>Turing Tip Jar</h1></Link>
-        <div>
-          <NavLink to='/module/1' activeClassName='nav-button'>
-            <button className="mod-button">Module 1</button>
-          </NavLink>
-          <NavLink to='/module/2' activeClassName='nav-button'>
-            <button className="mod-button">Module 2</button>
-          </NavLink>
-          <NavLink to='/module/3' activeClassName='nav-button'>
-            <button className="mod-button">Module 3</button>
-          </NavLink>
-          <NavLink to='/module/4' activeClassName='nav-button'>
-            <button className="mod-button">Module 4</button>
-          </NavLink>
-        </div>
+        {/* <div> */}
+          <NavLink to='/module/1' activeClassName='nav-button' className='mod-button'>Module 1</NavLink>
```

Comment:
Ah good, you converted these back to the just the Navlinks (not wrapping button elements). I think this is a good fix, seeing as how the Navlink styling was not contained inside of the button, so we were losing all the navlink benefits.